Formula/ollama-elastic.rb

```diff
@@ -0,0 +1,36 @@
+class OllamaElastic < Formula
+  desc ""
+  homepage ""
+  version "dbeaea4"
+
+  on_macos do
+    url "https://github.com/elastic/ollama/releases/download/dbeaea4/ollama-darwin"
+    sha256 "df241ff0b7f0f4ed3be36e2f9322848bd6b2a4dec205a84e0b104272a6393ffe"
+    def install
+      bin.install "ollama-darwin" => "ollama-elastic"
+    end
+  end
+
+  on_linux do
+    on_intel do
+      if Hardware::CPU.is_64_bit?
+        url "https://github.com/elastic/ollama/releases/download/dbeaea4/ollama-linux-amd64.tgz"
+        sha256 "fa8429dccf0f484aab143e934cc3fa2ffb2d07519c62b380d3554d12ed5af6ec"
+
+        def install
+          bin.install "bin/ollama" => "ollama-elastic"
+        end
+      end
+    end
+    on_arm do
```

Comment:
a little confused per "homebrew doesn't seem to support linux/arm64." should we still build?

Reply:
Figured it doesn't hurt to keep it here in case brew adds support out of band later. If it's confusing, can remove it. The build won't be faster unfortunately since it's the Darwin artifact that takes quite a while.

Reply:
makes sense